### ISSUE TITLE
{crash} ArangoDB 3.7.2-1 [linux], thread 0 caught unexpected signal 11 (SIGSEGV) accessing address 0x00000000000000b8: signal handler invoked

### ISSUE THREAD

**Issue by zz123happy** [arangodb/arangodb#12722]
## My Environment

* __ArangoDB Version__:        3.7.2-1
* __Storage Engine__:             RocksDB 
* __Deployment Mode__:        Cluster
* __Deployment Strategy__:   Manual Start in Docker 
* __Configuration__:               
* __Infrastructure__:               own 
* __Operating System__:        Linux version 3.10.0-1062.18.1.el7.x86_64 (mockbuild@kbuilder.bsys.centos.org) (gcc version 4.8.5 20150623 (Red Hat 4.8.5-39) (GCC) ) #1 SMP Tue Mar 17 23:49:17 UTC 2020
* __Total RAM in your machine__:     
* __Disks in use__:        
* __Used Package__:     

## Component, Query & Data

__arangod-dbserver-2.conf__:
```
# ArangoDB configuration file
#
# Documentation:
# https://www.arangodb.com/docs/stable/administration-configuration.html
#

[database]
directory = /var/lib/arangodb3

[server]
endpoint = tcp://0.0.0.0:8529
storage-engine = rocksdb
statistics = true
io-threads = 64
minimal-threads = 16

[javascript]
startup-directory = /usr/share/arangodb3/js
app-path = /var/lib/arangodb3-apps

[foxx]
# enable Foxx queues in the server
# queues = true

# interval (seconds) to use for polling jobs in Foxx queues
# queues-poll-interval = 1

[log]
level = trace
file = /var/log/arangodb3/arangod-dbserver.log

[rocksdb]
# encryption-keyfile=/your-encryption-file
#use-direct-reads = true
max-total-wal-size = 1024000
write-buffer-size = 2048000
max-write-buffer-number = 10
total-write-buffer-size = 81920000
#write-buffer-size = 67108864
#max-write-buffer-number = 15
#min-write-buffer-number-to-merge = 2
#table-block-size = 4096
enforce-block-cache-size-limit = true
block-cache-size = 8589934592

[cache]
size = 10485760
#8589934592

[network]
io-threads = 32
max-open-connections = 80000

[cluster]
my-address = tcp://10.73.11.100:6001
my-role = DBSERVER
agency-endpoint = tcp://10.73.14.189:8529,tcp://10.73.14.190:8529,tcp://10.73.14.194:8529

[replication]
active-failover = true
```
__docker-compose__:
```
  arangodb-dbserver-zz-2:
    image: registry.api.weibo.com/arangodb/arangodb:3.7.2
    container_name: "cluster-arangodb-dbserver-zz-2"
    network_mode: "bridge"
    cap_add:
      - SYS_NICE
    ports:
      - 6001:8529
    environment:
      - ARANGO_NO_AUTH=1
    volumes:
      - "/etc/localtime:/etc/localtime:ro"
      - "./arangod-dbserver-2.conf:/etc/arangodb3/arangod.conf"
      - "./dbserver-2:/var/lib/arangodb3"
      - "./apps:/var/lib/arangodb3-apps"
      - "./logs-2:/var/log/arangodb3"
  arangodb-dbserver-zz-3:
    image: registry.api.weibo.com/arangodb/arangodb:3.7.2
    container_name: "cluster-arangodb-dbserver-zz-3"
    network_mode: "bridge"
    cap_add:
      - SYS_NICE
    ports:
      - 6011:8529
    environment:
      - ARANGO_NO_AUTH=1
    volumes:
      - "/etc/localtime:/etc/localtime:ro"
      - "./arangod-dbserver-3.conf:/etc/arangodb3/arangod.conf"
      - "./dbserver-3:/var/lib/arangodb3"
      - "./apps:/var/lib/arangodb3-apps"
      - "./logs-3:/var/log/arangodb3"

```
__/tmp/init-log__:
```
2020-09-29T07:36:09Z [8] INFO [e52b0] ArangoDB 3.7.2-1 [linux] 64bit, using jemalloc, build tags/v3.7.2.1-0-g2c56dec4b2c-dirty, VPack 0.1.33, RocksDB 6.8.0, ICU 64.2, V8 7.9.317, OpenSSL 1.1.1g  21 Apr 2020
2020-09-29T07:36:09Z [8] INFO [75ddc] detected operating system: Linux version 3.10.0-1062.18.1.el7.x86_64 (mockbuild@kbuilder.bsys.centos.org) (gcc version 4.8.5 20150623 (Red Hat 4.8.5-39) (GCC) ) #1 SMP Tue Mar 17 23:49:17 UTC 2020
2020-09-29T07:36:09Z [8] INFO [25362] {memory} Available physical memory: 134588305408 bytes, available cores: 40
2020-09-29T07:36:09Z [8] TRACE [a1898] read failed for file '/proc/sys/net/ipv4/tcp_tw_recycle': No such file or directory
2020-09-29T07:36:09Z [8] WARNING [e8b68] {memory} /sys/kernel/mm/transparent_hugepage/enabled is set to 'always'. It is recommended to set it to a value of 'never' or 'madvise'
2020-09-29T07:36:09Z [8] WARNING [e8b68] {memory} /sys/kernel/mm/transparent_hugepage/defrag is set to 'always'. It is recommended to set it to a value of 'never' or 'madvise'
2020-09-29T07:36:09Z [8] WARNING [f3108] {memory} execute 'sudo bash -c "echo madvise > /sys/kernel/mm/transparent_hugepage/enabled"'
2020-09-29T07:36:09Z [8] WARNING [f3108] {memory} execute 'sudo bash -c "echo madvise > /sys/kernel/mm/transparent_hugepage/defrag"'
2020-09-29T07:36:09Z [8] DEBUG [63a7a] host ASLR is in use for shared libraries, stack, mmap, VDSO, heap and memory managed through brk()
2020-09-29T07:36:09Z [8] DEBUG [713c0] {authentication} Not creating user manager
2020-09-29T07:36:09Z [8] INFO [43396] {authentication} Jwt secret not specified, generating...
2020-09-29T07:36:09Z [8] DEBUG [71a76] {authentication} Setting jwt secret of size 64
2020-09-29T07:36:09Z [8] INFO [144fe] using storage engine 'rocksdb'
2020-09-29T07:36:09Z [8] INFO [3bb7d] {cluster} Starting up with role PRIMARY
2020-09-29T07:36:09Z [8] FATAL [a7902] {crash} ArangoDB 3.7.2-1 [linux], thread 0 caught unexpected signal 11 (SIGSEGV) accessing address 0x00000000000000b8: signal handler invoked
2020-09-29T07:36:09Z [8] INFO [308c3] {crash} frame 1 [0x00007f1987da424d] *no symbol name available for this frame
2020-09-29T07:36:09Z [8] INFO [308c3] {crash} frame 2 [0x00007f198ad34795] *no symbol name available for this frame
2020-09-29T07:36:09Z [8] INFO [ded81] {crash} available physical memory: 134588305408, rss usage: 71417856, vsz usage: 182829056, threads: 6
```

## Steps to reproduce

The same config for conf and docker-compose worked well in version 3.6.1 but failed in 3.7.2, nothing changed except the version .

__Expected result__:

help me to find the problem.

**Comment by dothebart** [comment]
Hi, 
we have a known issue with graph queries - is there one involved? 

If not, can you install a GDB 9 and the [debug symbols](https://www.arangodb.com/download-major/centos/) and launch the instance using gdb? https://github.com/arangodb/arangodb/blob/devel/README_maintainers.md#crashes

**Comment by maxkernbach** [comment]
Hi @zz123happy,

In the meantime version 3.7.3, which includes a fix for the graph query SegFault issue (#12507), has been released.

Is this crash still occurring with 3.7.3? If so, can you please use gdb as requested?

**Comment by Simran-B** [comment]
@zz123happy Can you check whether 3.7.3 fixes the problem?
